src/pages/gen2/build-a-backend/auth/set-up-auth/index.mdx

```diff
@@ -156,7 +156,7 @@ import type { WithAuthenticatorProps } from '@aws-amplify/ui-react';
 
 Amplify.configure(config);
 
-function App({ Component, pageProps, signOut, user }: WithAuthenticatorProps) {
+function App({ signOut, user }: WithAuthenticatorProps, { Component, pageProps }: AppProps) {
```

Comment:
This seems incorrect, React components do not provide props as separate positional parameters. Have you run this example code locally?

Reply:
thank you, updated this